core/app/models/spree/customer_return.rb

```diff
@@ -31,6 +31,10 @@ def fully_reimbursed?
       completely_decided? && return_items.accepted.includes(:reimbursement).all? { |return_item| return_item.reimbursement.try(:reimbursed?) }
     end
 
+    def currency
+      order.currency
```

Comment:
```suggestion
      order&.currency
```

order can be nil in some cases